<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBT_RFI_DATA_QUERY</class>
 <widget class="QMainWindow" name="GBT_RFI_DATA_QUERY">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GBT_RFI_DATA_QUERY</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>362</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>GBT RFI DATA QUERY</string>
    </property>
    <widget class="QListWidget" name="receivers">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>221</width>
       <height>192</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Receivers</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Date of
 Observation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDateEdit" name="target_date">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>221</width>
       <height>26</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>MM/dd/yy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="start_frequency">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="end_frequency">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Frequency
 (MHz)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>End Frequency
 (MHz)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="plot_button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>360</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="saveData">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>360</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save Data
 From Plot</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Use your
 Ctrl Button
 to select
 &gt;1 receiver</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
